appengine-managed-runtime/src/main/java/com/google/apphosting/vmruntime/VmRuntimeFileLogHandler.java

```diff
@@ -71,34 +69,18 @@ private static String fileLogPattern() {
     return directory + "/" + DEFAULT_LOG_PATTERN;
   }
 
-  /**
-   * Initialize the {@code VmRuntimeFileLogHandler} by installing it on the root logger.
-   */
-  public static void init() throws IOException {
-    reloadLoggingProperties(LogManager.getLogManager());
+  @Override
+  public void configure() throws IOException {
     Logger rootLogger = Logger.getLogger("");
+
+    // Look for the expected Handlers
     for (Handler handler : rootLogger.getHandlers()) {
```

Comment:
Fine as is. A java 8 alternative (which I am not convinced is more readable) could be:
```java
Optional<Handler> handler =
    rootLogger.getHandlers().filter(x -> x instance of VmRuntimeFileLogHandler).findAny();
if (!handler.isPresent()) {
  rootLogger.addHandler(new VmRuntimeFileLogHandler());
}